packages/lesswrong/components/users/NewUserCompleteProfile.tsx

```diff
@@ -161,6 +171,12 @@ const NewUserCompleteProfile: React.FC<NewUserCompleteProfileProps> = ({ current
       </div>}
       {/* TODO: Something about bio? */}
       <div className={classes.submitButtonSection}>
+        {forumTypeSetting.get() === "EAForum" &&
+          <Typography variant="body1" className={classes.sectionHelperText} gutterBottom>
+            I agree to the <TosLink />, including my content being available
+            under a <LicenseLink /> license
```

Comment:
```suggestion
            under a <LicenseLink /> license.
```

Match the punctuation on the rest of the page.